src/v2/BaseLoginRouter.js

```diff
@@ -95,6 +95,11 @@ export default Router.extend({
   },
 
   handleIdxResponseSuccess (idxResponse) {
+    //On success flush sessionStorage, 'success' object in remediation signifies end of login flow
+    if(idxResponse.rawIdxState.success || idxResponse.rawIdxState.successWithInteractionCode) {
+      sessionStorage.clear();
```

Comment:
seems too brutal since it could clear up other data from session storage that are set by other applications.

Reply:
Yup, @aarongranick-okta mentioned this in chat, i would just remove the item by name.